ikasaneip/configuration-service/src/main/java/org/ikasan/configurationService/metadata/JsonConfigurationMetaDataProvider.java

```diff
@@ -69,7 +69,9 @@ public ConfigurationMetaData deserialiseMetadataConfiguration(String metaData) {
         try
         {
             //JSON file to Java object
-            result = this.mapper.readValue(metaData, List.class);
+            //result = this.mapper.readValue(metaData, List.class);
```

Comment:
Why leave this as commented - remove the line

Reply:
Removed